python/cuspatial/cuspatial/core/binpreds/feature_overlaps.py

```diff
@@ -20,17 +23,18 @@ class OverlapsPredicateBase(EqualsPredicateBase):
     equals predicate for all implementations up to this point.
```

Comment:
"up to this point"? What does this mean?

Reply:
All implementations at this point in time. Will clarify.